modules/dnn/src/layers/resize_layer.cpp

```diff
@@ -64,6 +64,8 @@ class ResizeLayerImpl : public ResizeLayer
             outputs[0][2] = zoomFactorHeight > 0 ? (outputs[0][2] * zoomFactorHeight) : outHeight;
             outputs[0][3] = zoomFactorWidth > 0 ? (outputs[0][3] * zoomFactorWidth) : outWidth;
         } else {
+            CV_CheckGE(inputs.size(), (size_t)1, "");
```

Comment:
If `inputs.size()` can be 0(I assume it can), then it's already UB in line 62, otherwise this check is unnecessary.